game/scripts/vscripts/components/progression/hero_progression.lua

```diff
@@ -1,48 +1,73 @@
 if HeroProgression == nil then
     HeroProgression = class({})
     Debug.EnabledModules['progression:*'] = true
+
```

Comment:
Fixed the indentation on the entire file - been using 4 spaces before, not its 2 spaces. That's why the changes are so many.